src/service/chain_rewardable_entities.proto

```diff
@@ -24,30 +21,36 @@ message pub_key {
 enum mobile_hotspot_device_type {
   mobile_hotspot_device_type_UNKNOWN =
       0; // Unknown device type, should not be used
-  mobile_hotspot_device_type_WIFI_INDOOR = 1;
-  mobile_hotspot_device_type_WIFI_OUTDOOR = 2;
-  mobile_hotspot_device_type_WIFI_DATA_ONLY = 3;
+  mobile_hotspot_device_type_CBRS = 1;
+  mobile_hotspot_device_type_WIFI_INDOOR = 2;
+  mobile_hotspot_device_type_WIFI_OUTDOOR = 3;
+  mobile_hotspot_device_type_WIFI_DATA_ONLY = 4;
 }
 
 message mobile_hotspot_metadata {
-  // Timestamp when the NFT was created
-  uint64 created_at_ms = 1;
```

Comment:
would like to hold onto this - it's our "deployed_at" marker for downstream ETL. Is there perhaps a better place to put it?

Reply:
Hmm i think we've been using this as the marker for `deployed_at`: https://github.com/helium/proto/blob/master/src/service/mobile_config.proto#L80

Which seems potentially broken. Downstream ETL can likely maintain this in app DB and backfill with the current values we have. Though interesting that the existing values we've been using may not actually represent a deployment time.